helm/values-local.yaml

```
@@ -0,0 +1,281 @@
+        memory: "2Gi"
+        cpu: "1"
+
+# ── Kafka ─────────────────────────────────────────────────────────────────────
+
+kafkaConfig:
+  saslPassword: "flexprice123"
+
+kafka:
+  enabled: false  # disable Bitnami subchart; hand-rolled template uses confluentinc/cp-kafka
+
+kafkaConfig:
```

YAMLlint (1.38.0)

[error] duplication of key "kafkaConfig" in mapping

(key-duplicates)